package-lock.json

```diff
@@ -5404,12 +5404,14 @@
       "dependencies": {
```

Comment:
I don't see a change to package.json so I'm not sure why the lock file updated. I think we should remove this change before merging.